### DISCUSSION TITLE
Section Pattern when the Section title doesn't have a colon :

### DISCUSSION THREAD

**Discussion by kswamy15** [medspacy/medspacy#59] [Q&A]
I see how to write new section patterns like this:
new_patterns = [
    SectionRule(category="history_of_present_illness", literal="HPI:", 
            pattern=[{"LOWER": {"REGEX": "hpi"}}, {"LOWER": ":"}])
]
and this works.  But what I have a Chief Complaint text like this:
Chief Complaint   Patient presents with   • Follow Up       HPI/Interval History:     y.o.female returns

How to write the pattern to recognize the Chief Complaint section.  I have tried a few combinations like this but doesn't work unless I use the colon: and if the Chief Complaint has a colon: at the end of it in the clinical text.   

SectionRule(category="chief_complaint", literal="Chief Complaint ", 
            pattern=[{"LOWER": "chief complaint"},{"LOWER": " "}]),

**Comment by kswamy15** [comment]
I found the answer to my question - has to be written using spacy rule based matching using tokens.  So the pattern would be this as Chief and Complaint are two separate tokens.  

SectionRule(category="chief_complaint", literal="Chief Complaint", 
            pattern=[{"LOWER": "chief"},{"LOWER": "complaint"}]),

**Reply by abchapman93** [reply]
Hi! Yes, the solution you found will work. Alternatively, you can leave the pattern argument out and just use the literal, which will match the lower-cased phrase:
`SectionRule(category="chief_complaint", literal="Chief Complaint")`

Whenever possible I like to use the literal argument since it's simpler than the spaCy dictionary lists.
